indico/modules/events/editing/controllers/backend/management.py

```diff
@@ -157,4 +158,22 @@ def _process_PUT(self):
 
     def _process_DELETE(self):
         editable_type_settings[self.editable_type].set(self.event, 'self_assign_allowed', False)
+
+
+class RHEditableTypePrincipals(RHEditableTypeManagementBase):
+    def _process_GET(self):
+        principals = {p.principal for p in self.event.acl_entries
```

Comment:
`= [p.principal.identifier for ...]` and then just `return jsonify(principals)` below (or even merge everything in a single statement if it doesn't look bad)